tests/results/DiffTest.py

```diff
@@ -17,6 +17,11 @@ def test_add_lines(self):
         self.uut.add_lines(0, ['t'])
         self.uut.add_lines(0, [])
 
+    def test_add_line(self):
```

Comment:
Hey, I added that into test. And it returned the error again. :/
Did I write something wrong?